mteb/benchmarks/benchmarks.py

```diff
@@ -3,16 +3,34 @@
 from dataclasses import dataclass
 from typing import Sequence
 
+from pydantic import AnyUrl, BeforeValidator, TypeAdapter
+from typing_extensions import Annotated
+
 from mteb.abstasks.AbsTask import AbsTask
 from mteb.overview import get_tasks
 
+http_url_adapter = TypeAdapter(AnyUrl)
+STR_URL = Annotated[
```

Comment:
I would probably name it something like `StrUrl` or `StringUrl`, as all captial letters kinda suggest that it's somehow a constant.

Reply:
changed to UrlString